src/tasks/AndroidAppBuilder/Templates/monodroid.c

```diff
@@ -216,6 +223,21 @@ mono_droid_runtime_init (const char* executable, int managed_argc, char* managed
     appctx_values[0] = ANDROID_RUNTIME_IDENTIFIER;
     appctx_values[1] = bundle_path;
 
+    char* file_name = "output.pefile";
+    char file_path [1024];
+    int res1 = snprintf (file_path, sizeof (file_path) - 1, "%s/%s", bundle_path, file_name);
+    struct stat buffer;
+    
+    LOG_INFO ("file_path: %s\n", file_path);
+    assert (res1 > 0);
+
+    if (stat (file_path, &buffer) == 0) {
+        MonovmRuntimeConfigArguments *arg = (MonovmRuntimeConfigArguments *) malloc (sizeof (MonovmRuntimeConfigArguments));
+        arg->kind = 0;
+        arg->runtimeconfig.name.path = file_path;
```

Comment:
This works, but it's pretty fragile: `file_path` is allocated on the stack, but `monovm_runtimeconfig_initialize` will save it away in a global var and it will be used only later in `mono_jit_init_version`.  It will work as long as both of those calls are in this function - but will start pointing to garbage if this is ever refactored.

better to heap allocate `file_path` and free it from `cleanup_runtime_config`

Reply:
Fixed. Thanks!